terraform/module-lambda-roles/main.tf

```diff
@@ -48,7 +48,10 @@ data "aws_iam_policy_document" "payment_reconciler_execution" {
       "s3:GetObject",
       "logs:PutDestination",
       "logs:DescribeResourcePolicies",
-      "logs:DescribeDestinations"
+      "logs:DescribeDestinations",
+      "ec2:CreateNetworkInterface",
+      "ec2:DescribeNetworkInterfaces",
+      "ec2:DeleteNetworkInterface"
```

Comment:
While this can probably go in for now we should create a story to go back and review what is required specifically. The danger here is that other people will just copy and paste elsewhere